### ISSUE TITLE
Using `textAlignment` on `columns` results to type error

### ISSUE THREAD

**Issue by charisra** [icflorescu/mantine-datatable#172]
**Describe the bug**
I passed `textAlignment: 'right'` on one of my columns and got the following error

```
Types of property 'textAlignment' are incompatible.
        Type 'string' is not assignable to type 'DataTableColumnTextAlignment | undefined'.ts(2322)
DataTableProps.d.ts(57, 5): The expected type comes from property 'columns' which is declared here on type 'IntrinsicAttributes & ITableProps<any>'

```
**To Reproduce**
1. Use `textAlignment` in a column

**Expected behavior**
There should be no error when I use a string. That's exactly what's shown in the [official example docs](https://icflorescu.github.io/mantine-datatable/examples/column-properties), too.

**Screenshots**

**Desktop (please complete the following information):**
 - OS: OSX
 - Browser N/A
 - Version N/A

**Smartphone (please complete the following information):**

**Additional context**

**Comment by icflorescu** [comment]
This should definitely not happen.
Can you please provide a codesandbox reproducing the issue?

**Comment by russocorp** [comment]
> 

i´m using this.
import type { DataTableColumnTextAlignment } from 'mantine-datatable/dist/types/DataTableColumnTextAlignment';

**Comment by icflorescu** [comment]
Passing `'left' | 'center' | 'right'` to `textAlignment` definitely works, as shown in the official docs website, which is actually a "real" Next.js application.
That's why I'm asking for a codesandbox example, @charisra. Unless you provide it, I can't figure out what the problem might be in your specific setup.

@russocorp: I suggest you import the type like this directly from `mantine-datatable`:
```ts
import type { DataTableColumnTextAlignment } from 'mantine-datatable';
```

**Comment by charisra** [comment]
@icflorescu sorry it took me a while to get back to you.
Here's a [CodeSandbox](https://codesandbox.io/s/busy-hugle-u6phj1?file=/src/App.tsx). This seems like an actual issue in Typescript.

**Comment by icflorescu** [comment]
Yes, looks like a TS issue.
If you're planning to use it like that, you could make the error go away by typing the columns like that:

```ts
const tableColumns: DataTableColumn<{ submissionDate: string }>[] = [
  {
    accessor: "submissionDate",
    textAlignment: "right"
  }
];
```

**Comment by charisra** [comment]
That works, thanks. Though wondering if this needs to be fixed in the types so `textAlignment` is supported out of the box without such tricks.

**Comment by ark-davegoodchild** [comment]
I've just hit this same issue. I was not importing the type, and I've tried by importing the type, however I still get an error stating: ` Type 'string' is not assignable to type 'DataTableColumnTextAlignment | undefined'` I set the `textAlignment` to `undefined`, no error, I change it to any of the 3 options and the error returns. 

So far, I can't find any way around this.
```
const columns = [
  { name: 'code', accessor: 'code', title: 'Meter' },
  { name: 'avg-usage', accessor: 'avgDailyUsage', title: 'Avg. Daily Usage (kWh)', textAlignment: undefined },
  { name: 'total-usage', accessor: 'totalUsage', title: 'Total Usage (kWh)', textAlignment: 'right' }
]
```

I've ensured I'm running the latest version (2.9.1).

<img width="1108" alt="Screenshot 2023-07-26 at 10 11 01" src="https://github.com/icflorescu/mantine-datatable/assets/126799077/31af3b19-03aa-4976-bd96-b1d672c74fc1">
